### DISCUSSION TITLE
CUCM 12.5

### DISCUSSION THREAD

**Discussion by Letshadow** [levensailor/ciscoaxl#29] [Q&A] [answered]
I have tried to obtain information from a user in the CUCM of cisco labs DEVNET, I was not successful,

`ucm = axl('axlacces', 'Claro123', '10.10.20.1','12.5')`
`a=ucm.get_users("user18")`
`a`
![imagen_2022-05-30_150619941](https://user-images.githubusercontent.com/44853747/171054100-3e5e3df6-96de-492a-8e42-3962c05648a6.png)
investigating I found this

https://community.cisco.com/t5/management/cucm-12-5-passing-uuid-to-getline-or-listline-errors-out-but/m-p/3899541

would it affect this file if I understand correctly it affects this file
https://github.com/levensailor/ciscoaxl/blob/5631482c2fade4ea7de37f1f1728aa862b95016a/ciscoaxl/schema/12.5/AXLSoap.xsd

**Comment by bradh11** [comment, marked as answer]
@Letshadow - There is a method called `ucm.get_user()` in addition to `ucm.get_users()`.

To achieve what it looks like you are attempting to do (pull details on a single user), i would recommend using the `ucm.get_user()` method instead.   below is a known good working example (leveraging python-decouple library to pull in os environment variables):

```python
from ciscoaxl import axl
from decouple import config

cucm = config("CUCM_PUBLISHER")
username = config("CUCM_AXL_USERNAME")
password = config("CUCM_AXL_PASSWORD")
version = config("CUCM_VERSION")

ucm = axl(username=username, password=password, cucm=cucm, cucm_version=version)

get_user = ucm.get_user("admin")
print(get_user)


```

**Reply by Letshadow** [reply]
Oh thank you very much
